crates/nostr/src/nips/nip47.rs

```diff
@@ -50,15 +50,15 @@ impl std::error::Error for Error {}
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
-            Self::Key(e) => write!(f, "Key: {e}"),
-            Self::JSON(e) => write!(f, "Json: {e}"),
-            Self::Url(e) => write!(f, "Url: {e}"),
-            Self::Secp256k1(e) => write!(f, "Secp256k1: {e}"),
-            Self::NIP04(e) => write!(f, "NIP04: {e}"),
-            Self::UnsignedEvent(e) => write!(f, "Unsigned event: {e}"),
```

Comment:
Can you leave these? Are useful in bindings to understand well the errors (like in python, avoiding to try-except every line of code)